File: LE6/d6-1.java
import java.util.Scanner;

public class Ex6_V1 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        double valor, desconto;

        System.out.print("Digite o valor do produto: ");
        valor = sc.nextDouble();

        System.out.print("Digite o desconto (%): ");
        desconto = sc.nextDouble();

        double valor_final = valor - (valor * desconto / 100);

        System.out.println("Valor final com desconto: " + valor_final);
    }
}

File: RP6/d6-1.cpp
#include <iostream>
using namespace std;

int main() {
    double valor, desconto;

    cout << "Digite o valor do produto: ";
    cin >> valor;

    cout << "Digite o desconto (%): ";
    cin >> desconto;

    double valor_final = valor - (valor * desconto / 100);

    cout << "Valor final com desconto: " << valor_final << endl;

    return 0;
}